mslice/widgets/cut/cut.ui

```diff
@@ -34,7 +34,11 @@
       </widget>
      </item>
      <item row="0" column="3">
-      <widget class="QLineEdit" name="lneCutStart"/>
+      <widget class="QLineEdit" name="lneCutStart">
```

Comment:
I'm not sure about the place holder text. In the rest of mantid we generally use placeholder text for showing default values. I'm not sure this makes it much clearer as the `from`/`to` labels are quite good indicators of what you need to provide.

Reply:
Okay, I'll keep to that standard and remove them. Thank you.